src/SPC/builder/unix/library/attr.php

```diff
@@ -19,7 +19,7 @@ protected function build(): void
                 'LDFLAGS' => trim('-L' . BUILD_LIB_PATH . ' ' . $this->getLibExtraLdFlags()),
                 'LIBS' => $this->getLibExtraLibs(),
             ])->execWithEnv('./autogen.sh')
-            ->execWithEnv('./configure --prefix= --enable-static --disable-shared')
+            ->execWithEnv('./configure --prefix= --enable-static --disable-shared --with-pic --without-libintl-prefix --without-libiconv-prefix')
```

Comment:
see my other PR, I think that's the besser solution. disabling nls should prevent it from looking for intl or iconv.